### PR TITLE
10667 collapsable menu and additional layers toggles

### PR DESCRIPTION
**Pull request by horatiorosa**
## Summary
This PR adds a toggle for showing the outlines of two additional layers, NTA's and Community Districts, that are not present on the default map.

####  Acceptance Criteria
1) collapsable menu  button with additional layer toggles
2) NTA layer outline toggle-able 
3) CD* (PUMA) layer outline toggle-able

##### Additional Layer Menu
Screen Shot 2022-10-12 at 10 59 24 AM

##### NTA toggle selected
Screen Shot 2022-10-12 at 10 59 45 AM

##### Community District toggle selected
Screen Shot 2022-10-12 at 10 59 53 AM

#### Tasks/Bug Numbers
 - Fixes:
[AB#10667](https://dev.azure.com/NYCPlanning/cc280b0d-40a0-4689-b852-2e6247f1af50/_workitems/edit/10667)
[AB#10668](https://dev.azure.com/NYCPlanning/cc280b0d-40a0-4689-b852-2e6247f1af50/_workitems/edit/10668)
[AB#10669](https://dev.azure.com/NYCPlanning/cc280b0d-40a0-4689-b852-2e6247f1af50/_workitems/edit/10669)

### Technical Explanation
This PR adds an additional component, `AdditionalMapLayers.tsx` with toggles that passes a boolean to the map to show (or not) additional layers.

### CODE REVIEW

**Review by bmarchena — APPROVED** [review]
:rocket: :moon: :alien:

**Review by TylerMatteo — CHANGES_REQUESTED** [review]

File: src/components/AdditionalMapLayers/AdditionalMapLayers.tsx [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+                  Neighborhood Tabulation Area (NTA)
+                </Box>
+                <Box className="nta-square" justifySelf={"end"}>
+                  <LayerSelectionSquareNTA />
+                </Box>
+              </Box>
+            </FormLabel>
+          </FormControl>
+          <FormControl
+            className="cd-wrapper"
```
[the remark this answers is not in the document]
Comment (horatiorosa) [reply src/components/AdditionalMapLayers/AdditionalMapLayers.tsx:167]: it was to find the right wrappers and divs. They can get cut

File: src/hooks/useLayers/useLayers.ts [hunk 2]
```diff
@@ -257,5 +256,44 @@ export const useLayers = (
...
+      visible: ntaOutlineLayer,
+      type: MAP_TYPES.QUERY,
+      id: "unique_id_nta_outline",
+      data: `SELECT * FROM ${process.env.NTA_LAYER}`,
+      uniqueIdProperty: "id",
+      getLineColor: [114, 138, 238, 192],
+      getFillColor: [0, 0, 0, 0],
+      lineWidthUnits: "pixels",
+      getLineWidth: 3,
+      updateTriggers: {
```
[the remark this answers is not in the document]
Comment (horatiorosa) [reply src/hooks/useLayers/useLayers.ts]: still works! both are gone

File: src/pages/map/[view]/[geography].tsx [hunk 3]
```diff
@@ -229,13 +235,22 @@ const MapPage = ({ initialRouteParams }: MapPageProps) => {
 
       {view === "drm" && geoid && <DrmMobileDrawer />}
 
-      <Box flex="2" height="100%">
+      <Box flex="2" height="100%" className="thisAndThat">
```
[the remark this answers is not in the document]
Comment (horatiorosa) [reply src/pages/map/[view]/[geography].tsx:238]: yep

**Review by TylerMatteo — APPROVED** [review]

**Pull request merged**